BuildTasks/Pester.Task.ps1

```diff
@@ -1,34 +1,34 @@
 task Pester {
     $requiredPercent = $Script:CodeCoveragePercent
+    Import-Module Pester -Force
```

Comment:
we don't support pester 5+ so I can't really help you here, the help tests rely on pester 4 functionality, pester 5 appears to be completely throwing out variables from the outer scope within Context blocks. The tests should work fine on Pester 4.10 (they are still passing for me on https://github.com/loanDepot/SnowSQL master branch. I can't test 4.10 on your branch since your changes here will not run on 4.10 due to the changes below being 5.0 functionality).

I would recommend rolling the pester changes back, honestly, We've had nothing but trouble with pester 5 historically. Feel free to replace with a forced load of 
```Import-Module Pester -RequiredVersion 4.10.1 -Force```
and a silent install of 4.10.1 if needed on appveyor (not sure how that works, really).

PSScriptAnalyzer and Pester updates are pretty involved sometimes, and we normally bulk update all our modules at once to a new version of pester once we know what we're getting ourselves into rather than piecemealing like this. If you can figure out how to make Pester 5 work here, be my guest, but it looks like the $node variables are completely gone once you enter the Context block in Pester 5, so you'd have to work around that somehow. I'm sure there's a way if you want to figure it out, but since we upgrade Pester in bulk across 50+ modules by merging a common base module repo in, we'd rather just solve it there once and leave this working as-is on Pester 4.10.1 until then, and merge that in one big batch to all our modules as we normally do.